### PR TITLE
New UI: record analytics for UITips dialog choices

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

**Comment by breville** [comment]
cc @tanyaparker

File: apps/src/templates/studioHomepages/UiTips.jsx [hunk 1]
```diff
@@ -102,6 +107,8 @@ const UiTips = React.createClass({
       `/api/v1/users/${this.props.userId}/post_ui_tip_dismissed`,
       { tip: this.props.tipId }
     );
+
+    trackEvent("ui_tips", this.props.tipId, "before_dialog_cancel");
```
Comment (tanyaparker) [inline apps/src/templates/studioHomepages/UiTips.jsx:111]: Does cancel count both clicking the Skip button and the X?

Comment (breville) [reply apps/src/templates/studioHomepages/UiTips.jsx:111]: Yes.  We don't currently differentiate between those two.  We could if desired.

Comment (tanyaparker) [reply apps/src/templates/studioHomepages/UiTips.jsx:111]: Nope, that's all we need. Thanks!!

File: apps/src/templates/studioHomepages/UiTips.jsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ import React from 'react';
 import UiTip from './UiTip';
 import Dialog from '../Dialog';
 import _ from 'lodash';
+import trackEvent from '../../util/trackEvent';
```
Comment (islemaster) [inline apps/src/templates/studioHomepages/UiTips.jsx:13]: :+1:

**Pull request merged**